includes/components/class-attribute.php

```diff
@@ -980,6 +1003,13 @@ public function add_edit_fields( $form_args, $form ) {
 			}
 		}
 
+		// Set attributes default values.
+		foreach ( $model->_get_fields() as $field) {
+			if ( ! $field->get_value() && $field->get_default() ) {
+				$field->set_value( $field->get_default() );
```

Comment:
This is possibly affected by another bug, it was a part of the task "adding an option to disable number formatting", please check if the fix in that task resolves this because I assume that setting defaults for fields will pre-fill them on the back-end (in meta boxes) even if you clear the field and save changes